python_on_whales/components/node/models.py

```diff
@@ -75,7 +75,7 @@ class NodeStatus(DockerCamelModel):
 
 
 class NodeManagerStatus(DockerCamelModel):
-    leader: bool
+    leader: Optional[bool]
```

Comment:
Thank you for the pull request! I already encountered this kind of error. This is why I made a decorator `all_fields_optional`, but it seems that I forgot to use it on `node/models.py`. Can you use it? You can see an exemple in this file: https://github.com/gabrieldemarmiesse/python-on-whales/blob/master/python_on_whales/components/container/models.py

Reply:
Thank you for the quick response! I have updated the pull request to use the decorator you suggested.